### PR TITLE
add start and stop scripts; all projects same version

### PR DESCRIPTION
**Pull request by dturanski**
see README

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,21 +1,24 @@
-/application.yml
```
Comment (markfisher) [inline .gitignore:1]: Did you intentionally overwrite `.gitignore`? There are some entries here that should not be deleted. For example, I am now seeing `.factorypath` files.

Comment (dturanski) [inline .gitignore:1]: No, not intentional, but needed to add some things. I'll take a look

**Comment by markfisher** [comment]
Otherwise LGTM and works great! I'll merge as soon as you update the `.gitignore` file.

**Comment by markfisher** [comment]
squashed and merged as https://github.com/springone2gx2015/vehicle-fleet-demo/commit/0140975be4193fc00941d3ba558e30f86430989f

**Pull request closed without merging**